modules/network/front-door-web-application-firewall-policy/main.bicep

```diff
@@ -0,0 +1,141 @@
+metadata name = 'Front Door Web Application Firewall (WAF) Policies'
+metadata description = 'This module deploys a Front Door Web Application Firewall (WAF) Policy.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Front Door WAF policy.')
+@minLength(1)
+@maxLength(128)
+param name string
+
+@description('Optional. Location for all resources.')
+param location string = 'global'
+
+@allowed([
+  'Standard_AzureFrontDoor'
+  'Premium_AzureFrontDoor'
+])
+@description('Optional. The pricing tier of the WAF profile.')
+param sku string = 'Premium_AzureFrontDoor'
```

Comment:
Should this default to standard?

Reply:
This was from the original contributor. No clue. There seem to be a few differences such as the lack of ManagedRule definitions: https://learn.microsoft.com/en-us/azure/frontdoor/standard-premium/tier-comparison . Can't say I have much of an opinion about this one without knowing the service better.

Reply:
Updated it now and will re-run the pipeline

Reply:
Yes it is fine to use standard as default. I have added conditional deployment for ManagedRule section